src/lib/connectors/repos/queries/billing-invoices.js

```diff
@@ -47,3 +47,9 @@ where
   and t2.original_billing_invoice_id is not null
 returning *;
 `;
+
+exports.resetIsFlaggedForRebillingByInvoiceId = `
```

Comment:
you will also need to update the rebilling state for the invoice as well as the original invoice id column.

If the original_invoice_id === invoice_id then this means it is the first rebill in a rebilling chain then set rebilling_status = null and original_invoice_id = null
otherwise set the rebilling_state = 'rebill' and leave the original_invoice_id as is